include/zephyr/drivers/video-controls.h

```diff
@@ -31,42 +38,91 @@ extern "C" {
 
 /**
  * @name Control classes
+ *
+ * List of control classes, to group related controls together in blocks.
+ *
+ * This list is kept identical to the Linux kernel definitions.
+ *
  * @{
  */
-#define VIDEO_CTRL_CLASS_GENERIC	0x00000000	/**< Generic class controls */
-#define VIDEO_CTRL_CLASS_CAMERA		0x00010000	/**< Camera class controls */
-#define VIDEO_CTRL_CLASS_MPEG		0x00020000	/**< MPEG-compression controls */
-#define VIDEO_CTRL_CLASS_JPEG		0x00030000	/**< JPEG-compression controls */
-#define VIDEO_CTRL_CLASS_VENDOR		0xFFFF0000	/**< Vendor-specific class controls */
+#define VIDEO_CTRL_CLASS_BASE			0x00980900 /**< Base class controls */
```

Comment:
Same here :

```
#define V4L2_CTRL_CLASS_USER	0x00980000
#define V4L2_CID_BASE			(V4L2_CTRL_CLASS_USER | 0x900)
```

Reply:
The "base" class already did contain the `BASE` keyword. Would I rename it into `VIDEO_CID_BASE`?
Along with the other `VIDEO_CTRL_CLASS_..._BASE` renamed to `VIDEO_CID_..._BASE`?
It is a quick change if that needs be.

Reply:
Glad that this exception got checked still!